docs/concepts/gui-components.mdx

```diff
@@ -0,0 +1,243 @@
+---
+title: "GUI Components"
+description: "Customize how your agents appear and behave in the BeeAI platform interface"
+---
+
+GUI components in the BeeAI platform define how your agents appear and behave in the user interface. Through the `PlatformUIAnnotation` metadata, you can customize the visual presentation and interaction patterns for your agents.
+
+<Note>
+To fully customize your agent's appearance and behavior in the interface, you'll be using metadata defined by the Agent Communication Protocol (ACP). For details on how metadata annotations work, see the [ACP metadata documentation](https://agentcommunicationprotocol.dev/core-concepts/agent-manifest#schema-metadata).
+</Note>
+
+## Configuration
+
+The `PlatformUIAnnotation` is the primary configuration object that defines how your agent appears in the GUI. It's specified within your agent's metadata annotations.
```

Comment:
@jenna-winkler this seems a bit off here - because as a reader it's hard to me understand why Configuration is anyhow different from Display Properties. 

The idea here is that `PlatformUIAnnotation` is extension of ACP to serve things that are not part of the protocol, while the other section (Display Properties) covers standardized parts of the protocol. They both do serve for definition of how agent appear in the GUI.

The way this is documented here just makes me wonder why configuration is to define how agent appears in GUI when the display properties are doing effectively the same and why they are separated.